mumerge.py

```diff
@@ -852,14 +882,17 @@ def bed_line_formatter(chromosome, mu_sig_list, width=1.0):
             
             # Status counter and update at stdout
             if verbose:
-                sys.stdout.write("\rProcessed {} of {} regions"
+                sys.stdout.write("\nProcessed {} of {} regions"
                                    .format(count, total))
                 count += 1
 
             # Select Tfit calls for one region
             mu_list = tfit_dict[chromosome][region]
 #            print(chromosome, region, (region[0]+region[1])/2, mu_list)
-
+            if call_remover(mu_list):
+                if verbose:
+                    sys.stdout.write("\nskipping singleton...")
```

Comment:
Do you want this to be printed every iteration?

The verbose flag is intended to print stuff to the command line, and not necessarily log print statements for each iteration (that's what the log file is for, if needed)